src/components/App.jsx

```diff
@@ -34,7 +38,27 @@ const KeyboardAwareSidebar = ({ children }) => {
 
 const App = props => {
   const { showBottomNav, settingsCollection } = props
+  const { t } = useI18n()
+  const client = useClient()
   const settings = getDefaultedSettingsFromCollection(settingsCollection)
+
+  const showAlert = t => {
```

Comment:
why `t` is passed as an argument ?

Reply:
Because it is used in show Alert. `t('Error.fetch-error')` and `t('General.reload')`

Reply:
But you already have access to `t` within this scope, no? What is the gain to have in passed in args?

Reply:
Well seen